### ISSUE TITLE
Set up Sass

### ISSUE THREAD

**Issue by coka** [tari-project/block-explorer-frontend#3]
The process of adding Sass to an app that uses `react-scripts` is outlined [here](https://create-react-app.dev/docs/adding-a-sass-stylesheet/).

`SideBar.css` looks especially amenable to Sass features. We can also extract colors to their own module.

**Comment by coka** [comment]
@Krakaw Let's flag this as a good first issue?

**Comment by coka** [comment]
Do you guys still want to use Sass?

**Comment by Krakaw** [comment]
It would be a nice to have.

**Comment by c-franklin** [comment]
If this is still open I can take it.

**Comment by delta1** [comment]
Thanks @charlottefranklin go ahead

**Comment by c-franklin** [comment]
> Thanks @charlottefranklin go ahead

@delta1 is there a back-end I can connect to by changing an env file in the front-end vs setting up the entire back-end locally? Have installed scss and written functions for colours (so you can theme) but need to ensure I'm testing everything before I PR. -- Have joined the #tari-dev IRC @pentacle

**Comment by delta1** [comment]
Let me see what I can do @charlottefranklin - I will ping you on IRC. My handle there is also @delta1

**Comment by delta1** [comment]
@charlottefranklin please use the following `.env` file for your testing 

```
REACT_APP_EXPLORER_API_DOMAIN=explore.tari.com/api
REACT_APP_EXPLORER_API_PREFIX=https
REACT_APP_EXPLORER_WS_PREFIX=ws
REACT_APP_TOKEN_NAME=tXTR
```

**Comment by c-franklin** [comment]
PR in https://github.com/tari-project/block-explorer-frontend/pull/114

**Comment by stringhandler** [comment]
Fixed in #114

**Comment by delta1** [comment]
Thanks @charlottefranklin !
